src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -76,6 +76,28 @@ String getTimestampString() {
     return timestampString;
   }
 
+  // given a dominion style csv json file path:
+  // read associated manifest data
+  // read Dominion cvr json into CastVoteRecords
+  // write CastVoteRecords to generic cvr csv files: one per contest
+  // return true if files are successfully written
+  // return false if an error is encountered
+  public static Boolean convertDominionCvrJsonToGenericCsv(String dominionCvrJsonFile) {
+    Boolean success = true;
```

Comment:
Is there a particular reason we're returning a boolean here instead of making this `void` like `convertToCdf()`? I'd lean towards copying that design pattern if not.

Reply:
yeah, it's a better api.  The caller should have a way of knowing if it succeeded or not.

Reply:
nit: boolean, not Boolean

Reply:
In that case, should we file and issue to do this for `convertToCdf()` and `tabulate()` as well?